playground/frontend/playground_components/test/src/controllers/example_loaders/common.dart

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:playground_components/playground_components.dart';
+
+import 'package:playground_components/src/controllers/example_loaders/example_loader.dart';
+import 'package:playground_components/src/controllers/example_loaders/example_loader_factory.dart';
+
+class TestExampleLoadingDescriptor extends ExampleLoadingDescriptor {
+  final Sdk? sdk;
+  final bool ok;
```

Comment:
I think it should be named more specifically

Reply:
`+`